src/Html.php

```diff
@@ -210,11 +211,13 @@ public static function cleanParametersURL($url)
      **/
     public static function resume_text($string, $length = 255)
     {
-        if (Toolbox::strlen($string) > $length) {
-            $string = Toolbox::substr($string, 0, $length) . "&nbsp;(...)";
+        $append = '';
+        if (mb_strlen($string, 'UTF-8')) {
```

Comment:
```suggestion
        if (mb_strlen($string, 'UTF-8') > $length) {
```

I don't know if you removed this check on purpose but without it any non empty string will be appended `&nbsp;(...)`.